modin/core/storage_formats/pandas/native_query_compiler.py

```diff
@@ -0,0 +1,1270 @@
+    dt_freq = _register_default_pandas(
+        lambda df: pandas.DataFrame([df.squeeze(axis=1).dt.freq])
+    )
+    dt_hour = _register_default_pandas(_dt_prop_map("hour"))
+    dt_is_leap_year = _register_default_pandas(_dt_prop_map("is_leap_year"))
+    dt_is_month_end = _register_default_pandas(_dt_prop_map("is_month_end"))
+    dt_is_month_start = _register_default_pandas(_dt_prop_map("is_month_start"))
+    dt_is_quarter_end = _register_default_pandas(_dt_prop_map("is_quarter_end"))
+    dt_is_quarter_start = _register_default_pandas(_dt_prop_map("is_quarter_start"))
+    dt_is_year_end = _register_default_pandas(_dt_prop_map("is_year_end"))
+    dt_is_year_start = _register_default_pandas(_dt_prop_map("is_year_start"))
+    dt_microsecond = _register_default_pandas(_dt_prop_map("microsecond"))
+    dt_microseconds = _register_default_pandas(_dt_prop_map("microseconds"))
+    dt_minute = _register_default_pandas(_dt_prop_map("minute"))
+    dt_month = _register_default_pandas(_dt_prop_map("month"))
+    dt_month_name = _register_default_pandas(_dt_func_map("month_name"))
+    dt_nanosecond = _register_default_pandas(_dt_prop_map("nanosecond"))
+    dt_nanoseconds = _register_default_pandas(_dt_prop_map("nanoseconds"))
+    dt_normalize = _register_default_pandas(_dt_func_map("normalize"))
+    dt_quarter = _register_default_pandas(_dt_prop_map("quarter"))
+    dt_qyear = _register_default_pandas(_dt_prop_map("qyear"))
+    dt_round = _register_default_pandas(_dt_func_map("round"))
+    dt_second = _register_default_pandas(_dt_prop_map("second"))
+    dt_seconds = _register_default_pandas(_dt_prop_map("seconds"))
+    dt_start_time = _register_default_pandas(_dt_prop_map("start_time"))
+    dt_strftime = _register_default_pandas(_dt_func_map("strftime"))
+    dt_time = _register_default_pandas(_dt_prop_map("time"))
+    dt_timetz = _register_default_pandas(_dt_prop_map("timetz"))
+    dt_to_period = _register_default_pandas(_dt_func_map("to_period"))
+    dt_to_pydatetime = _register_default_pandas(_dt_func_map("to_pydatetime"))
+    dt_to_pytimedelta = _register_default_pandas(_dt_func_map("to_pytimedelta"))
+    dt_to_timestamp = _register_default_pandas(_dt_func_map("to_timestamp"))
+    dt_total_seconds = _register_default_pandas(_dt_func_map("total_seconds"))
+    dt_tz = _register_default_pandas(
+        lambda df: pandas.DataFrame([df.squeeze(axis=1).dt.tz])
+    )
+    dt_tz_convert = _register_default_pandas(_dt_func_map("tz_convert"))
+    dt_tz_localize = _register_default_pandas(_dt_func_map("tz_localize"))
+    dt_week = _register_default_pandas(_dt_prop_map("week"))
+    dt_weekday = _register_default_pandas(_dt_prop_map("weekday"))
+    dt_weekofyear = _register_default_pandas(_dt_prop_map("weekofyear"))
+    dt_year = _register_default_pandas(_dt_prop_map("year"))
+    duplicated = _register_default_pandas(pandas.DataFrame.duplicated)
+    eq = _register_default_pandas(_register_binary("eq"), filter_kwargs=["dtypes"])
```

Comment:
Why do we need to filter some kwargs?

Reply:
filtered dtypes in certain binary operators as `dtypes`, [could not be handled by pandas](https://github.com/arunjose696/modin/blob/arun-sqc/modin/core/dataframe/algebra/default2pandas/default.py#L124),

 But now I realize it is better to remove the case to case filtering for functions and adding dtypes to [exclude_names](https://github.com/arunjose696/modin/blob/arun-sqc/modin/core/storage_formats/pandas/native_query_compiler.py#L529) in `_register_default_pandas`, so removing filter_kwargs from arguments to function.